cmd/agent/app/integrations.go

```diff
@@ -40,6 +42,54 @@ var (
 	tufConfig    string
 )
 
+type integrationVersion struct {
+	major int
+	minor int
+	fix   int
+}
+
+func parseVersion(version string) (*integrationVersion, error) {
+	var major, minor, fix int
+	if version == "" {
+		return nil, nil
+	}
+	_, err := fmt.Sscanf(version, "%d.%d.%d", &major, &minor, &fix)
+	if err != nil {
+		return nil, fmt.Errorf("unable to parse version string %s: %v", version, err)
+	}
+	return &integrationVersion{major, minor, fix}, nil
+}
+
+func (v *integrationVersion) String() string {
+	return fmt.Sprintf("%d.%d.%d", v.major, v.minor, v.fix)
+}
+
+func (v *integrationVersion) isAbove(otherVersion *integrationVersion) bool {
```

Comment:
Could we rename this from `isAbove` to `isAboveOrEqualTo`? The latter name is more accurate about what the function actually does.